includes/author-filter/class-author-filter.php

```diff
@@ -0,0 +1,143 @@
+<?php
+/**
+ * Adds a filter by author to the Posts page in the Admin dashboard.
+ *
+ * @package Newspack
+ */
+
+/**
+ * Author Filter class
+ */
+class Author_Filter {
```

Comment:
I saw the PR earlier but forgot to bring it up. We should either put this class behind the `Newspack` namespace or a name prefix.

Reply:
oh, you are totally right, I missed that! will create a new PR

Reply:
Good idea! ➕